conference.js

```diff
@@ -157,13 +166,13 @@ function createInitialLocalTracksAndConnect(roomName) {
     return Promise.all([ tryCreateLocalTracks, connect(roomName) ])
         .then(([tracks, con]) => {
             APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
-            if (audioAndVideoError) {
+            if (audioAndVideoError || noVideo) {
                 if (audioOnlyError) {
                     // If both requests for 'audio' + 'video' and 'audio' only
                     // failed, we assume that there is some problems with user's
                     // microphone and show corresponding dialog.
                     APP.UI.showDeviceErrorDialog(audioOnlyError, null);
-                } else {
+                } else if (!noVideo) {
```

Comment:
the line below assumes that video error happened, but it's impossible with 'noVideo'